src/Tickets/Commerce/Gateways/PayPal/Gateway.php

```diff
@@ -155,4 +166,62 @@ public function render_checkout_template( \Tribe__Template $template ): string {
 
 		return $template->template( $template_path, tribe( Buttons::class )->get_checkout_template_vars() );
 	}
+	
+	/**
+	 * Filter to add any admin notices that might be needed.
+	 * 
+	 * @since TBD
+	 * 
+	 * @param array Array of admin notices.
+	 * 
+	 * @return array
+	 */
+	public function filter_admin_notices( $notices ) {
+		
+		// Check for unsupported currency.
+		$selected_currency = tribe_get_option( TC_Settings::$option_currency_code );
+		if ( $this->is_enabled() && ! $this->is_currency_supported( $selected_currency ) ){
+			$notices[] = [
+				'tc-stripe-currency-not-supported',
```

Comment:
should the notice name be neutral, or reference paypal?

Reply:
Doh! Got them backwards. LOL!